git/hooks/pre-push

```diff
@@ -6,6 +6,15 @@ source "${REPO_DIR}/util/logging.sh"
 source "${REPO_DIR}/util/path.sh"
 WPTD_PATH=${WPTD_PATH:-$(absdir "${REPO_DIR}")}
 
+
+# Check for uncommitted changes.
+! git diff . 2>&1 | read > /dev/null 2>&1
+DIFF_STATUS="${?}"
+if [ "${DIFF_STATUS}" != "0" ]; then
+  confirm "You have uncommitted local changes. Push committed changes anyway?"
+  if [ "${?}" != "0" ]; then exit ${?}; fi
```

Comment:
nit: Final `exit` in this script quotes the return code. I don't care whether it gets quoted (it's always a number), but we should make the file consistent.

Reply:
Quoted.